package.json

```diff
@@ -22,7 +22,8 @@
     "react-dom": "^18.3.1",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.5",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "web3": "^4.16.0"
```

Comment:
`web3` is not used in this PR. I suggest to remove it and remove `yarn.lock` changes, as well.